### DISCUSSION TITLE
圣遗物词条模型的修正（长期工作）

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#784] [General]
感觉小数值的分数还是太高了一些，我有时间去 NGA 抄几个模型综合算算
<img src="https://user-images.githubusercontent.com/9006264/161673401-029d1c34-d796-4ad1-bdb5-92f9e5651bae.jpg" width="200" />

```
全部词条：5.2
有效词条：4.8
--------------------
防御力：0.4
元素充能效率：2.6
攻击力：1.2
元素精通：1.0
```
<img src="https://user-images.githubusercontent.com/9006264/161673405-ad0a504e-33fc-433f-9e4f-fddc47f41488.jpg" width="200" />

```
全部词条：6.6
有效词条：3.8
--------------------
生命值：0.8
防御力：2.0
元素充能效率：1.0
攻击力：2.8
```

**Comment by Arondight** [comment]
这个是我随手填的，我就地算一下吧

就拿提瓦特第一大 c 迪卢克来说，满级 335 + 608 ，那么平均一下按照人均 900 攻击计算。

1. 大攻击 4.08% ，一个词条就是 36.72
2. 小攻击一个词条固定 13.62

小词条按照 0.37 折算比较合适，考虑到 900 略低，可以直接算 0.35

**Reply by Arondight** [reply]
我用上面的折算带入了一下实际情况，39 小防御是 1.4 词条，1.4 词条的大防御是 7.14%，女仆 800 防，7.14% 就是 57.12，还是差了点，感觉得分开计算一下折算

**Comment by Arondight** [comment]
我简单改了一下，现在应该结果精确一些了 https://github.com/Arondight/Adachi-BOT/commit/44a1ad914643801e27a0b797bf4bc94a725996ac

**Reply by Arondight** [reply]
<img src="https://user-images.githubusercontent.com/9006264/161673401-029d1c34-d796-4ad1-bdb5-92f9e5651bae.jpg" width="200" />

```
您的空之杯（水元素伤害加成）词条数为：
全部词条：4.8
有效词条：4.4
--------------------
防御力：0.3
元素充能效率：2.6
攻击力：0.9
元素精通：1.0
```

<img src="https://user-images.githubusercontent.com/9006264/161673405-ad0a504e-33fc-433f-9e4f-fddc47f41488.jpg" width="200" />

```
您的空之杯（水元素伤害加成）词条数为：
全部词条：6.4
有效词条：3.8
--------------------
生命值：0.6
防御力：2.0
元素充能效率：1.0
攻击力：2.8
```

**Comment by mark9804** [comment]
这个先对付一下，我到时候平平淡淡建个模吧，不过也不会精确多少估计，人都不一样

**Reply by Arondight** [reply]
这个没法建模，角色那么多，建模也没意义，我感觉还不如挑个典型了，例如小攻击就按照五星c算，班尼特的小攻击那当然不管了
